### PR TITLE
RDCC-2077: Adding env variables and S2s authorised services.

### PR DESCRIPTION
**Pull request by nitishsingla1234**
### JIRA link (if applicable) ###
Not Applicable

### Change description ###
Adding env variables and S2s authorised services.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ idam:
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
     pbaFromUserAccountMap: false
   s2s-authorised:
-    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend}
+    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend,am_org_role_mapping_service}
   api.url: ${idam_url:https://idam-api.aat.platform.hmcts.net}
```
Comment (shashank-rastogi) [inline src/main/resources/application.yaml:89]: am_org_role_mapping_service - is not live now so it cant be added to application yaml.. we need to add this to flux config and enable this in lower env only
@AdilOoze  - can u add this to flux config for aat and demo?

Comment (AdilOoze) [reply src/main/resources/application.yaml:89]: @shashank-rastogi sure, I will add `am_org_role_mapping_service` to flux config for `aat` and `demo`

Comment (AdilOoze) [reply src/main/resources/application.yaml:89]: @shashank-rastogi here is the Pull Request for the above: https://github.com/hmcts/cnp-flux-config/pull/7601

**Comment by nitishsingla1234** [comment]
This PR is not required.

**Pull request closed without merging**